### ISSUE TITLE
Funding Roadmap

### ISSUE THREAD

**Issue by afroleft** [DoESLiverpool/covid19#81]
We just hit our stretch goal of `£20,000` 🍾

* Production looks to be up and roaring
* Orders and fulfilment processing now a full-time job
* Delivery drivers are up and down the M62
* Partner producers are getting orders over the line too 

@mjamjoum is gathering who does / does not want paying on a spreadsheet. Back of the card calculations are showing looking like we can pay peeps for 2/3weeks. After that we're on volunteer fumes and any materials people might donate. 

* We have Liverpool Heart & Chest wanting a weekly `2000` rolling order #59. 
* My former ceo from [Patchwork](https://www.patchwork.health) wants to link with us with their network of hospitals so we can reach more healthcare workers in need. 
* We have a `10,000` order from Lancashire CCG. 
* And many more

To fulfil these orders it will cost money.

---

## @hannah-forbes asks this earlier 

> Now we’ve hit our target do you just want a soft refresh - the message being “check out our new campaign video”? OR do we want to push for a new target and make these videos part of a new big push?

> My thoughts are that with lots of other campaigns going live we won’t be able to get a huge amount more organically (maybe another 5k) but if you did want to do another push these videos will of course be a great opportunity.

> Let me know your thoughts - I plan on updating the video on the campaign page today (unless there are any objections) but would appreciate your input on the key messages for planning social media.

---

## A proposal

1. We try to define our unit cost, so that we can say for every £X donated, we are getting 10 visors in to the hands of healthcare workers 
2. We continue to _gently_ peddle the overfund with the message above + new Campaign Vids #78 . Leave the target at £20k though 
3. We push for corporate donations 
4. @hannah-forbes attracts more exposure through other audiences such as Independent Liverpool etc 
5. We also have councils wanting to give us money @hannah-forbes can unpack. Personally don't know an awful lot about the soft money space, but it's always an option 

With the extra funds: 
* We can pay for more materials
* Buy new machinery & tooling 
* Consumables 
* Pay for fuel and time of delivery drivers
* Keep the operation fully staffed for longer 

More visors out the door seems to make sense with the information I currently have to hand.   

Feedback welcome. Not precious about any of it, just want the best ideas to win.

**Comment by hannah-forbes** [comment]
To add further thoughts:

I think it's vital we have further detail on what the money is being spent on (and how if at all this has changed with the increase). I think since this is the second increase in target we should clarify why we’re increasing the target again, why we need the new amount and why we’re now able to settle on this new amount.

While I hope no one would take this stance some may find the increase opportunistic and may prefer further clarity for the increase.

**Comment by ajlennon** [comment]
>We have a 10,000 order from Lancashire CCG.
>To fulfil these orders it will cost money.

What's current capacity?

**Comment by afroleft** [comment]
> I think since this is the second increase in target we should clarify why we’re increasing the target again

As set out above, not proposing to change the target for now @hannah-forbes:

> We continue to gently peddle the overfund with the message above + new Campaign Vids #78 . **Leave the target at £20k though**

But the explanation for continuing the raise is: 

> With the extra funds:
> We can pay for more materials
> Buy new machinery & tooling
> Consumables
> Pay for fuel and time of delivery drivers
> Keep the operation fully staffed for longer

Plus, we have huge orders coming through (see above) that will cost money to fulfil.

**Comment by hannah-forbes** [comment]
Sure, completely agree with the approach for overfunding. So I can just say "check out our new video" etc. on social media. 

If we do decide to increase the target we can consider the above in more detail.

**Comment by afroleft** [comment]
> So I can just say "check out our new video" etc. on social media.

Deffo. The refreshed videos aren't coupled to this.

**Comment by mjamjoum** [comment]
Finance update:

I have created forms to be completed by staff. I have sent out these forms to @afroleft and @amcewen as they will be set out a line managers. All individuals who want to be paid will have to complete these forms (they should be uploaded as well to the main page so they can be easily pulled). In addition an expense form has been completed.

In addition, only staff working underneath @amcewen are expected to be paid. I have created a master form for him to note who is working. It will automatically calculate the amount owed to date (very easy to populate).

Anyone who is to be paid must complete the form, must get it approved by Tom or Adrienne. Once the form has been approved it can be sent to covidliverpoolfinance@gmail.com.

Any questions please let me know.

Thank you,
Mazen

**Comment by drakard** [comment]
We decided in #64 we will not be charging any money. I noticed there has also been a concern about running out of money to provide the most help possible.

Is there a possibility of another funding model, were those suppliers who wish to make large orders (for example 10,000 units) can purchase material themselves (with our help of course), which we then manufacture for them for free? Doing that, DoES would not be charging for anything, but it would greatly reduce costs and spin out the funds we have available (potentially indefinitely). It could potentially be an optional model for those who feel they can afford it.
